sermant-agentcore/sermant-agentcore-config/config/plugins.yaml

```diff
@@ -7,6 +7,7 @@
 # 'plugins' is used to configure static plugins. The plugins take effect when Sermant starts by premain. Plugins that support static installation are configured and cannot be uninstalled
 plugins:
   - flowcontrol
```

Comment:
no need to change

Reply:
The header parameter must be set before the router-plugin. Otherwise, routing cannot effect.